ProjectSettings/SceneTemplateSettings.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
What's the purpose of this new file?

Reply:
I believe, but am not confidant, that when unity makes a new scene it need to know what dependencies it needs to import into the scene and how to import them. This is things like the animator for play mode and physic modules and whether to import things like textures as a completely new texture or as a reference to a texture in another scene. I suspect that we may need to talk about project settings soon and make everyone has the same settings and the right one for this game

Reply:
We can try removing it but I suspect it will come back again and again and if they are different between people that may have some issues down the line.

Reply:
Thoughts?